src/index.ts

```diff
@@ -104,9 +106,43 @@ function isPrimitive(p: Property | Parameter): boolean {
   return !!p.$ref.enum || !p.$ref.properties;
 }
 
-function isDateTime(p: Property | Parameter | null): boolean {
-  if (!p) return false;
-  return p.type === "string" && p.format === "date-time";
+export type XtpTyped = { xtpType: XtpNormalizedType };
```

Comment:
this is an external convenience type for any node in the doc that should have an xtp type. think schema, property, parameter, etc.